### PR TITLE
docs/add week of refine index page

### PR DESCRIPTION
**Pull request by salihozdemir**
Added a new page to list the week of refine series on documentation.

### CODE REVIEW

**Review by alicanerdurmaz — CHANGES_REQUESTED** [review]

File: documentation/src/pages/week-of-refine/index.tsx [hunk 1]
```diff
@@ -1,8 +1,119 @@
...
+                        <div
+                            className={clsx(
+                                "flex flex-col landing-xl:flex-row",
+                                "gap-10 landing-xl:gap-0",
+                                "blog-sm:mt-10 blog-sm:mb-20",
+                            )}
+                        >
+                            <RefineWeekLogoXL className="flex-shrink-0 landing-xl:mr-[116px]" />
+                            <p className="text-[32px] leading-10 font-semibold text-gray-800 dark:text-gray-100">
+                                Lorem ipsum dolor sit amet, consectetuer
```
Comment (alicanerdurmaz) [inline documentation/src/pages/week-of-refine/index.tsx:99]: @necatiozmen do we have text for that ?

**Review by alicanerdurmaz — CHANGES_REQUESTED** [review]

File: documentation/src/pages/week-of-refine/constants.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { CardProps } from ".";
+import { StrapiWithText, SupabaseWithText } from "./icons";
```
[the remark this answers is not in the document]
Comment (salihozdemir) [reply documentation/src/pages/week-of-refine/constants.ts:2]: I saw these icons, but when I tried to use them, I realized that they are not suitable for the design of the page. Existing icons are smaller than the ones I created. Also, I tried to give width and height to the existing icons, but they were not affected. So, I created new icons.

File: documentation/src/pages/week-of-refine/icons/strapi.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
...776 8.6823 39.1025 9.33295C39.1321 9.37838 39.1898 9.39678 39.2393 9.37475L42.072 8.11728C42.1142 8.09857 42.1308 8.04739 42.1073 8.00765C42.0761 7.95488 42.035 7.88598 42.0077 7.8429C41.5221 7.07706 40.8941 6.50653 40.1064 6.07186C39.2388 5.59308 38.2759 5.35369 37.2174 5.35369C35.812 5.35369 34.6364 5.72366 33.6908 6.46359C32.7452 7.20353 32.2723 8.21767 32.2723 9.50603C32.2723 10.3591 32.5022 11.0686 32.962 11.6344C33.4219 12.2003 33.9814 12.6137 34.6408 12.8749C35.3001 13.136 35.9594 13.3537 36.6188 13.5278C37.2782 13.7019 37.8377 13.8978 38.2975 14.1153C38.7574 14.333 38.9872 14.6159 38.9872 14.9641C38.9872 15.7128 38.3886 16.0871 37.1914 16.0871C36.0517 16.0871 35.262 15.6645 34.8222 14.8193C34.7844 14.7467 34.698 14.7135 34.6232 14.7467L31.7984 16.0026C31.7586 16.0202 31.7412 16.0673 31.7601 16.1065C31.7715 16.1302 31.7836 16.155 31.7924 16.1727C32.7795 18.1526 34.5792 19.1426 37.1914 19.1426C38.6836 19.1426 39.9285 18.7813 40.9262 18.0588C41.9239 17.3363 42.4228 16.3047 42.4228 14.9641C42.4228 14.0762 42.1929 13.3363 41.7331 12.7443C41.2733 12.1524 40.7137 11.7302 40.0544 11.4778C39.395 11.2253 38.7356 11.0164 38.0763 10.8509C37.4169 10.6856 36.8574 10.494 36.3976 10.2765Z"
```
[the remark this answers is not in the document]
Comment (salihozdemir) [reply documentation/src/pages/week-of-refine/icons/strapi.tsx:16]: It is not compressed, I gonna compress it now.

**Review by alicanerdurmaz — APPROVED** [review]

**Review by yildirayunlu — APPROVED** [review]

File: documentation/src/assets/week-of-refine/constants.ts [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+        bgLinearGradient:
+            "bg-week-of-refine-strapi-card-light dark:bg-week-of-refine-strapi-card",
+        link: "/week-of-refine-strapi",
+    },
+    {
+        title: "Pixels!",
+        imgURL: "https://refine.ams3.cdn.digitaloceanspaces.com/week-of-refine/week-of-refine-pixels.png",
+        dateRange: "February 14 - February 20, 2023",
+        description:
+            "Stay tuned all week and learn the basics of creating a CRUD app via refine & Strapi on a fun project in a week.",
```
Comment (necatiozmen) [inline documentation/src/assets/week-of-refine/constants.ts:21]:
```suggestion
            "Stay tuned all week and learn the basics of creating a CRUD app via refine & Supabase on a fun project in a week.",
```

File: documentation/src/assets/week-of-refine/constants.ts [hunk 5]
```diff
@@ -0,0 +1,27 @@
+import { CardProps } from "@site/src/pages/week-of-refine";
+import { StrapiWithText, SupabaseWithText } from "../integration-icons";
+
+export const weekOfRefineCards: CardProps[] = [
+    {
+        title: "Invoicer",
+        imgURL: "https://refine.ams3.cdn.digitaloceanspaces.com/week-of-refine/week-of-refine-invoicer.png",
+        dateRange: "April 10 - April 14, 2023",
+        description:
+            "Stay tuned all week and learn the basics of creating a CRUD app via refine & Strapi on a fun project in a week.",
```
Comment (necatiozmen) [inline documentation/src/assets/week-of-refine/constants.ts:10]:
```suggestion
            "We'll be building an Invoice generator internal tool via refine & Strapi in a week.",
```

File: documentation/src/refine-theme/common-header/constants.ts [hunk 6]
```diff
@@ -99,12 +99,12 @@ export const MENU_ITEMS: MenuItemType[] = [
...
                 icon: ContributeIcon,
             },
-            // {
-            //     label: "refineweeks",
-            //     description: "Weekly projects to practice refine.",
-            //     link: "/week-of-refine",
-            //     icon: RefineWeekIcon,
-            // },
+            {
+                label: "refineweeks",
```
Comment (necatiozmen) [inline documentation/src/refine-theme/common-header/constants.ts:103]:
```suggestion
                label: "refineWeeks",
```

File: documentation/src/pages/week-of-refine/index.tsx [hunk 7]
```diff
@@ -1,8 +1,121 @@
...
+        </Link>
+    );
+};
 
 const RefineWeek = () => {
-    return <Redirect to="/" />;
+    return (
+        <CommonLayout>
+            <div className="not-prose">
+                <Head title="Week of refine | refine">
```
Comment (necatiozmen) [inline documentation/src/pages/week-of-refine/index.tsx:76]:
```suggestion
                <Head title="refineWeek | refine">
```

**Review by necatiozmen — APPROVED** [review]

**Pull request merged**